packages/react-components/src/components/Popover/Popover.module.scss

```diff
@@ -5,6 +5,7 @@
   display: none;
   max-width: 336px;
   min-width: 168px;
+  z-index: 1000;
```

Comment:
Could we use a dedicated variable from
https://github.com/livechat/design-system/blob/39fcca44937242ba3bd3bc7c8b855269d335b2f1/packages/react-components/src/utils/StackingContextLevel.scss

I believe we would need to introduce a new level:

W could follow this concept from LiveChat's Agent App:

```ts
export enum StackingContextLevel {
  Tooltip = 1000,
  Dropdown = 2000,
  ...
}
```